schema/2023-03/Form.json

```diff
@@ -325,7 +330,8 @@
         "SELECT",
         "SUBMISSION_REFERENCE",
         "TEXT",
-        "USERNAME"
+        "USERNAME",
+        "ACCORDION"
```

Comment:
enum is sorted alphabetically
move to top?

Reply:
Oh, yea thanks ill move it